### PR TITLE
SEAB-5183: created notebooks page and added to sitemap

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
Created notebooks page `dockstore.org/notebooks` and added the link to the sitemap.

Notebooks page
Screenshot from 2023-01-31 16-36-09

Sitemap
Screenshot from 2023-01-31 12-16-22

**Review Instructions**
Verify the new page renders and is implemented correctly and can be accessed from the sitemap.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-5183

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

**Comment by hyunnaye** [comment]
Will need to wait for https://github.com/dockstore/dockstore/pull/5327 to be merged for the tests to pass

File: src/app/workflows/workflows.component.html [hunk 1]
```diff
@@ -17,7 +17,12 @@
   <div fxLayout="row" fxLayoutAlign="start center">
     <img src="../assets/svg/sub-nav/{{ (entryPageTitle$ | async | titlecase) === 'Services' ? 'services.svg' : 'workflow.svg' }}" />
     <span>&nbsp;{{ entryPageTitle$ | async | titlecase }}&nbsp;</span>
-    <div *ngIf="(entryPageTitle$ | async | titlecase) === 'Services'" class="beta-label">preview</div>
+    <div
+      *ngIf="(entryPageTitle$ | async | titlecase) === 'Services' || (entryPageTitle$ | async | titlecase) === 'Notebooks'"
```
Comment (coverbeck) [inline src/app/workflows/workflows.component.html:21]: It was like this, so you don't have to change it, but it's a little fragile to rely on the display text for comparison -- if we ever tweak the title, this risks breaking.

You could add `this.entryType$ =this.sessionQuery.entryType$;` to workflow.component.ts, and then change this to `if (entryType$ | async == EntryType.Services || ....` (untested)

**Review by coverbeck — COMMENTED** [review]
One part optional, but I have the question about the image.

File: src/app/workflows/workflows.component.html [hunk 2]
```diff
@@ -17,7 +17,12 @@
   <div fxLayout="row" fxLayoutAlign="start center">
     <img src="../assets/svg/sub-nav/{{ (entryPageTitle$ | async | titlecase) === 'Services' ? 'services.svg' : 'workflow.svg' }}" />
```
[the remark this answers is not in the document]
Comment (hyunnaye) [reply src/app/workflows/workflows.component.html:18]: Yes, it needs to be updated. I'll look for some icons that can be used for notebooks.

[the remark this answers is not in the document]
Comment (hyunnaye) [reply src/app/workflows/workflows.component.html]: Added a placeholder image (https://fontawesome.com/icons/book?s=solid&f=classic) and set the color to `#3f51b5` which is the first Accent 1 colour in zeplin palette https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/5fc699d068c46f17551f745b

**Review by denis-yuen — COMMENTED** [review]
check the checklist, probably didn't run into anything but worth double checking

**Review by svonworl — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

File: src/app/workflows/workflows.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (coverbeck) [reply src/app/workflows/workflows.component.html]: > Added a placeholder image (https://fontawesome.com/icons/book?s=solid&f=classic) and set the color to `#3f51b5` which is the first Accent 1 colour in zeplin palette https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/5fc699d068c46f17551f745b

By placeholder, do you mean temporary? If yes, can you please create a followup ticket. 

If no, would it be possible to use the fontawesome image that we already bundle, like [here](https://github.com/dockstore/dockstore-ui2/blob/249f1d34f191ae27c4987dd055fa497047ee297f/src/app/container/versions/versions.component.html#L77). That way we don't have to bundle another copy, and it stays in sync/style with the other fontawesome icons if we upgrade the library. The downside is that you would use different HTML for this one case, and maybe we don't want the icon to stay in sync with the other fontawesome icons, we want it to stay in sync with the workflows/services SVGs, so I'm not sure.

**Review by kathy-t — APPROVED** [review]

File: src/app/workflows/workflows.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (hyunnaye) [reply src/app/workflows/workflows.component.html]: Created ticket https://github.com/dockstore/dockstore/issues/5343

**Review by coverbeck — APPROVED** [review]

**Pull request merged**